pyne/ensdf.py

```diff
@@ -68,6 +67,106 @@ def _to_id(nuc):
         return 0
     return nucid
 
+time_conv_dict = {'as': 1e-18,
+                  'attosec': 1e-18,
+                  'attosecond': 1e-18,
+                  'attoseconds': 1e-18,
+                  'fs': 1e-15,
+                  'femtosec': 1e-15,
+                  'femtosecond': 1e-15,
+                  'femtoseconds': 1e-15,
+                  'ps': 1e-12,
+                  'picosec': 1e-12,
+                  'picosecond': 1e-12,
+                  'picoseconds': 1e-12,
+                  'ns': 1e-9,
+                  'nanosec': 1e-9,
+                  'nanosecond': 1e-9,
+                  'nanoseconds': 1e-9,
+                  'us': 1e-6,
+                  'microsec': 1e-6,
+                  'microsecond': 1e-6,
+                  'microseconds': 1e-6,
+                  'ms': 1e-3,
+                  'millisec': 1e-3,
+                  'millisecond': 1e-3,
+                  'milliseconds': 1e-3,
+                  's': 1.0,
+                  'sec': 1.0,
+                  'second': 1.0,
+                  'seconds': 1.0,
+                  'm': 60.0,
+                  'min': 60.0,
+                  'minute': 60.0,
+                  'minutes': 60.0,
+                  'h': 3600.0,
+                  'hour': 3600.0,
+                  'hours': 3600.0,
+                  'd': 86400.0,
+                  'day': 86400.0,
+                  'days': 86400.0,
+                  'y': 86400.0*365.25,
+                  'year': 86400.0*365.25,
+                  'years': 86400.0*365.25,
+                  }
+
+
+# Energy to half-life conversion:  T1/2= ln(2) × (h/2 pi) / energy
+# See http://www.nndc.bnl.gov/nudat2/help/glossary.jsp#halflife
+# NIST CODATA https://physics.nist.gov/cgi-bin/cuu/Value?hbar
+#    h-bar = 1.054 571 800(13) x 1e-34 J
+#    1 J = 6.241 509 126(38) x 1e18 eV
+HBAR_LN2 = 4.5623775832376968e-16  # h-bar ln(2) in eV s
+energy_conv_dict = {'ev': HBAR_LN2,
+                    'kev': 1e-3 * HBAR_LN2,
+                    'mev': 1e-6 * HBAR_LN2,
+                    }
+
+
+def _to_sec(value, err, units):
```

Comment:
Also, I may recommend a name change here (`_to_sec_with_err()`?) since the interface no longer matches the other tosec function and it also converts energy half-lives.